packages/hash/frontend/src/pages/[shortname]/types/entity-type/[entity-type-id].page/use-entity-type-value.tsx

```diff
@@ -19,10 +25,47 @@ import {
   useFetchEntityTypes,
 } from "../../../../../shared/entity-types-context/hooks";
 
+interface EntityTypeAndPropertyTypes {
+  entityType: EntityType;
+  propertyTypes: Record<string, PropertyType>;
+}
+
+const getPropertTypes = (
```

Comment:
Typo
```suggestion
const getPropertyTypes = (
```

Reply:
Fixed typo in ac19ecbff4056f0befa28dc47a0762441553bb0b